react/week-2/lesson.md

```diff
@@ -169,7 +169,7 @@ Let's look at how a component is re-rendered ([interactive version](https://code
 function Counter(props) {
   console.log(`Rendering. props.count is ${props.likeCount}`);
 
-  return <button id="like-button">Likes: {props.likeCount}</button>;
+  return <button class="like-button">Likes: {props.likeCount}</button>;
```

Comment:
className here

Reply:
omg I can't believe I made this mistake!